softdevice_controller/include/sdc_hci_cmd_info_params.h

```diff
@@ -0,0 +1,645 @@
+/*
```

Comment:
should the header file be the same as the abbreviation perhaps?

`sdc_hci_cmd_ip.h`?

Reply:
I considered that. However, "info_params" is easier to understand and matches the spec. The user will understand the abbreviation used.